connect/1.3/README.md

```diff
@@ -104,6 +104,11 @@ This environment variable is recommended. Use this to set the JVM options for th
 
 This environment variable is optional. Use this to set the level of detail for Kafka's application log written to STDOUT and STDERR. Valid values are `INFO` (default), `WARN`, `ERROR`, `DEBUG`, or `TRACE`."
 
+### `DEBEZIUM_ENABLE_APICURIO`
+
+This environment variable is optional. Use this to enable [Apicur.io](https://www.apicur.io/) converters with 
```

Comment:
COuld you clarify whether this only applies to the existing Debezium connectors or also any other connectors users might add in derived images?

Reply:
it applies to all connectors in `$KAFKA_CONNECT_PLUGINS_DIR` (/kafka/connect/). the env var name is just a namespace to identify we provide and maintain it?
Like in my last comment in the JIRA ticket, should I maybe change this to `$CONNECT_PLUGIN_DIR`? when someone is using that image for something which is not Debezium in `$KAFKA_CONNECT_PLUGINS_DIR` and just sets `CONNECT_PLUGIN_DIR` it will not be added. found that after the initial work.

Reply:
> it applies to all connectors in $KAFKA_CONNECT_PLUGINS_DIR

Ok, would be good to mention that in the README then.

> Like in my last comment in the JIRA ticket, should I maybe change this to $CONNECT_PLUGIN_DIR? when someone is using that image for something which is not Debezium in $KAFKA_CONNECT_PLUGINS_DIR and just sets CONNECT_PLUGIN_DIR it will not be added. found that after the initial work.

Good question, looks indeed a bit irregular. Let's track it separately, though, could you log it in Jira? Thx!

Reply:
I just updated the PR to use `$CONNECT_PLUGIN_DIR` in a proper way. Kafka Connect `plugin.dir` can be a comma-separated list of directories. It now applies to all **active** connectors that are in use by Kafka Connnect.
Sorry I realized it some time after the initial PR creation.